lib/sarif/osv/base_sarif.rb

```diff
@@ -0,0 +1,54 @@
+require 'salus/bugsnag'
+
+module Sarif::OSV
+  class BaseSarif < Sarif::BaseSarif
+    include Salus::SalusBugsnag
+
+    OSV_URI = "https://osv.dev/list".freeze
+    SCANNER_NAME = "OSV Scanner".freeze
+
+    def initialize(scan_report, repo_path = nil)
+      super(scan_report, {}, repo_path)
+      @uri = OSV_URI
+      @logs = parse_scan_report!
+    end
+
+    def parse_scan_report!
+      logs = []
+      advisories = @scan_report.to_h.dig(:logs).split("\n\n")
```

Comment:
Are you parsing plain text here?  Is there a way to get the logs to show up as json or something?